src/test/java/teammates/test/pageobjects/InstructorFeedbackResultsPage.java

```diff
@@ -599,4 +605,40 @@ private String getElementSrcWithRetryAfterWaitForPresence(By by) {
         }
     }
 
+    /**
+     * Expands a particular panel, causing its results to load.
+     * @param panelNumber the position of panel element
+     */
+    public void loadResultPanel(int panelNumber) {
```

Comment:
Currently it is only used for question but in future can be used for other panels. Should we rename it ?